kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java

```diff
@@ -95,14 +93,14 @@ public void updateTopic() {
     @CaseId(207)
     @Test
     public void deleteTopic() {
-        pages.openTopicsList(CLUSTER_NAME)
+        topicsList.goTo(CLUSTER_NAME)
```

Comment:
we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla